packages/experimental/Menu/src/MenuItemCheckbox/useMenuItemCheckbox.ts

```diff
@@ -14,7 +14,33 @@ import { useMenuListContext } from '../context/menuListContext';
 const defaultAccessibilityActions = [{ name: 'Toggle' }];
 
 export const useMenuItemCheckbox = (props: MenuItemCheckboxProps): MenuItemCheckboxState => {
-  // attach the pressable state handlers
+  const { name } = props;
+  const context = useMenuListContext();
+  const checked = context.checked?.[name];
```

Comment:
What's with the brackets around [name] ?

Reply:
We want to access the property with the name that's the value of the name variable, not a property name.

i.e. if name is itemOne, we want to access `checked.itemOne`, which `checked.[name]` lets us do, not `checked.name`